bitcoin/src/network/constants.rs

```diff
@@ -141,6 +143,34 @@ impl Network {
         };
         Ok(network)
     }
+
+    /// Return the network's chain hash (genesis block hash).
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use bitcoin::network::constants::Network;
+    /// use bitcoin::blockdata::constants::ChainHash;
```

Comment:
```suggestion
    /// use bitcoin::constants::ChainHash;
    /// use bitcoin::Network;
```

Reply:
cc https://github.com/rust-bitcoin/rust-bitcoin/issues/1490 I think we can fix this when we deal with the typos issue